add filter_tabular 3.3.0; preserve filter_tabular 2.0.0

eschen42 opened the pull request:
@bgruening @hexylena 

This PR is to add `filter_tabular` to `tools_iuc.yaml`

Regarding the additions to `tools_iuc.yaml.lock`
`filter_tabular` 2.0.0 (revision 34d29339abab) is presently installed on UseGalaxy.eu
`filter_tabular` 3.3.0 (revision 4d5aae46f850) is to be installed onto UseGalaxy.eu

Thank you.

P.S. When I ran
```make fix```
it identified a number of issues that I did not include in this PR because it did not concern this tool.

bgruening commented on tools_iuc.yaml.lock:
```diff
@@ -8362,6 +8362,12 @@ tools:
+- name: filter_tabular
+  owner: iuc
+  revisions:
+  - 4d5aae46f850
```
do you need both version? can we just take the latest?

bgruening replied:
ping @eschen42

eschen42 commented:
@bgruening I removed the filter_tabular 2.0.0 version (which is presently installed on usegalaxy.eu) and left only the 3.3.0 version (which I need), assuming that this won't uninstall the old version and break others' workflows.  I also added v0.1.18 of galaxyp mqppep_anova and mqppep_preproc *without* removing older versions (because scripts/update-tool.py doesn't do that).

Is this what you expect?

Thank you.

bgruening reviewed (approved):
Yes, this looks correct.

eschen42 commented:
> Yes, this looks correct.

Thank you @bgruening
